packages/client-sdk-nodejs/src/internal/grpc/retry-interceptor.ts

```diff
@@ -25,6 +25,10 @@ export function createRetryInterceptorIfEnabled(
   ];
 }
 
+function addJitter(whenToRetry: number): number {
+  return (1 + Math.random()) * (1.2 * whenToRetry) - 1.1 * whenToRetry;
+}
+
```

Comment:
slightly different formula as the previous one was returning negative milliseconds

Reply:
The original was suggested by Chris:

```typescript
const jitter = Math.random() * (1.2 * delay) - (1.1 * delay)
```

I saw it was producing negative outputs so I added a  `+ 1`:

```typescript
const jitter = (1 + Math.random()) * (1.2 * delay) - (1.1 * delay)
```

But the goal was to add between -10% to +10% of jitter to the delay value. 
I think there's probably some more tweaking to do to the formula on second glance. Perhaps modifying your simplified version like so?

```typescript
const jitter = (0.2 * Math.random() + 0.9) * whenToRetry
```

Reply:
that last one looks promising. -10% to + 10% is definitely the goal.